integreat_cms/cms/forms/regions/region_form.py

```diff
@@ -185,6 +208,10 @@ def __init__(self, *args, **kwargs):
         self.fields["deepl_midyear_start_enabled"].initial = bool(
             self.instance.deepl_midyear_start_month
         )
+        self.disabled_offer_options = OfferTemplate.objects.filter(
+            pages__region=self.instance,
+        )
```

Comment:
```suggestion
        self.disabled_offer_options = (
            OfferTemplate.objects.filter(pages__region=self.instance)
            if self.instance.id
            else OfferTemplate.objects.none()
        )
```
Ah ok, then maybe like this? :thinking:

Reply:
Interesting, probably we can remove all `if self.instance` checks in the region form since all they're doing is checking whether the instance is a correct region objects and has the attributes we expect. But after calling `super().__init__(*args, **kwargs)` on the model form, the instance will never be `None`... I hope I'm not missing something important...